### ISSUE TITLE
both string and binary is not working with map_blocks

### ISSUE THREAD

**Issue by yupbank** [databricks/tensorframes#147]
so string datatype is not acceptable to tensorframes' tensor
and also binary can't work with map_blocks only with map_rows.

**Comment by thunterdb** [comment]
Yes, this is because string and byte arrays are variable length types and
in tensorflow do not seem to benefit so far from a vectorized API. This
could be revisited, I am sure tensorflow has improved in that respect.

Tim
